DatabaseTemplate/Modules/create-sk-template.psm1

```diff
@@ -0,0 +1,137 @@
+
+    $config.testHarnessJsonConfigLEAs = @(255901, 1, 2, 3, 4, 5, 6, 7, 6000203)
+    $config.testHarnessJsonConfig = "$PSScriptRoot\testHarnessConfiguration.SK.json"
+
+    $config.bulkLoadMaxRequests = 1
+    $config.schemaDirectories = @(
+        (Get-RepositoryResolvedPath "Application\EdFi.Ods.Standard\Artifacts\Schemas\")
+        ("$(Get-PluginFolderFromSettings $config.appSettings)\EdFi.Ods.Extensions.Sk*\Artifacts\Schemas\")
+    )
+
+    $config.databaseBackupName = "EdFi.Ods.Populated.Template.SK"
+    $config.packageNuspecName = "EdFi.Ods.Populated.Template.SK"
+
+    return $config
+}
+
+Set-Alias -Scope Global initpop Initialize-PopulatedTemplate
+function Initialize-SKTemplate {
+    <#
+    .SYNOPSIS
+        Creates a new Populated Template.
+
+    .DESCRIPTION
+        By default this will:
+        * Validate all xml files
+        * Resets the admin and security database
+        * Creates a new database for the populated template data to be loaded into
+        * Restores packages and build the bulk load client
+        * Copies sample files to isolate the files into two sections one for each of the two load scenarios
+        * Generates two apiclients with key/secret for the two necessary claimsets
+        * Starts the test harness api
+        * Executes first load scenario using the bootstrap data and claimset
+        * Executes second load scenario using the rest of the sample data and the sandbox claimset
+        * Stops the test harness api
+        * Creates a backup of the new populated template at: Ed-Fi-ODS-Implementation\DatabaseTemplate\Database\Populated.Template.bak
+        * Creates a .nuspec file for the new populated template at: Ed-Fi-ODS-Implementation\DatabaseTemplate\Database\Populated.Template.nuspec
+
+    .PARAMETER samplePath
+        An absolute path to the folder to load samples from, for example: C:\MySampleXmlData\.
+        Also supports specific version folders of the Data Standard repository, for example: C:\Ed-Fi-Standard\v3.0\ or C:\Ed-Fi-Standard\v2.0\
+
+    .PARAMETER noExtensions
+        Ignores any extension sources when running the sql scripts against the database.
+
+    .PARAMETER noValidation
+        Disables xml validation.
+
+    .parameter Engine
+    The database engine provider, either 'SQLServer' or 'PostgreSQL'
+
+    .EXAMPLE
+        PS> Initialize-PopulatedTempalate -samplePath "C:\edfi\Ed-Fi-Standard\v3.2\"
+    #>
+    param(
+        [Parameter(
+            Mandatory = $false,
+            HelpMessage = "An absolute path to the folder to load samples from, for example: C:\MySampleXmlData\.`r`nAlso supports specific version folders of the Data Standard repository, for example: C:\Ed-Fi-Standard\v3.0\ or C:\Ed-Fi-Standard\v2.0\"
+        )]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript( { Resolve-Path $_ } )]
+        [string] $samplePath = "$PSScriptRoot/../../../Starter-Kit-SEA-Data-Collection/sample-data/",
```

Comment:
and adjust this path for the starter kit repo like this
```suggestion
        [string] $samplePath = "$PSScriptRoot/Samples/Sample XML",
```

Reply:
updated